pep-0674.rst

```diff
@@ -16,9 +16,9 @@ Incompatible C API change disallowing using macros as l-value to:
 * Ease the C API implementation on other Python implementation;
 * Help migrating existing C extensions to the HPy API.
 
-On the PyPI top 5000 projects, only 14 projects (0.3%) are affected by 4
-macro changes. Moreover, 24 projects just have to regenerate their
-Cython code to use ``Py_SET_TYPE()``.
+On the PyPI top 5000 projects, only 13 projects (0.3%) are affected by 2
```

Comment:
```suggestion
Only 13 out of the top 5000 projects on PyPI (0.3%) are affected by 2
```

Reply:
There is a minor grammar and repetition issue here, but @vstinner here's a more conservative change if you prefer something closer to the original wording (@AA-Turner 's suggestion is fine too):

```suggestion
Of the top 5000 PyPI projects, only 13 (0.3%) are affected by 2
```

It might be helpful to clarify the "top 5000 _most downloaded_" given there can be different measures of "top"-ness, but perhaps I doth protest too much

```suggestion
Of the top 5000 most-downloaded PyPI projects, only 13 (0.3%) are affected by 2
```